src/07_slices.py

```diff
@@ -12,26 +12,28 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[0:1])
 
 # Output the second-to-last element: 9
-print()
+print(a[(len(a)-2):(len(a)-1)])
 
 # Output the last three elements in the array: [7, 9, 6]
-print()
+print(a[(len(a)-3):len(a)])
 
```

Comment:
You can do these just using numbers, no need for `len()` calls.  Line 32 is a good example of you doing just that.